go.sum

```diff
@@ -399,8 +399,8 @@ github.com/yuin/goldmark v1.1.32/go.mod h1:3hX8gzYuyVAZsxl0MRgGTJEmQBFcNTphYh9de
 github.com/yuin/goldmark v1.2.1/go.mod h1:3hX8gzYuyVAZsxl0MRgGTJEmQBFcNTphYh9decYSb74=
 github.com/yuin/goldmark v1.3.5/go.mod h1:mwnBkeHKe2W/ZEtQ+71ViKU8L12m81fl3OWwC1Zlc8k=
 github.com/yuin/goldmark v1.4.13/go.mod h1:6yULJ656Px+3vBD8DxQVa3kxgyrAnzto9xy5taEt/CY=
-go.mongodb.org/atlas v0.23.2-0.20230323121922-76f1516f3ff9 h1:5/GxsRQFvAW9UZD/GAcjQMQAfEWTjvdv1INYjqoL53w=
-go.mongodb.org/atlas v0.23.2-0.20230323121922-76f1516f3ff9/go.mod h1:r5HHjwxout7CJwjkAcJt+c2yA43rrB6LXidPk3pyeuo=
+go.mongodb.org/atlas v0.23.2-0.20230328111225-e40463c98c4e h1:Uy+DI9phtg3/M28DYH6u8Qvhh033yUhYlWlUqI2x2Iw=
+go.mongodb.org/atlas v0.23.2-0.20230328111225-e40463c98c4e/go.mod h1:QNr5VBLDcbEVnWRYrp3fTynE00Wad5jDT53lq5/q92s=
```

Comment:
Happy to merge these changes back to the vbranch provided we stick with the same sdk v2 version, so let's revert these changes and run a quick smoke test to validate before merging